src/DssSpell.t.sol

```diff
@@ -383,17 +349,61 @@ contract DssSpellTest is DssSpellTestBase {
         }
     }
 
-    function testOSMs() public skipped { // add the `skipped` modifier to skip
-        address READER = address(0);
+    function testOSMs() public { // add the `skipped` modifier to skip
```

Comment:
> Whitelist MkrOsm to read from current PIP_MKR using `DssExecLib.addReaderToWhitelist` with the following parameters:

[This part](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=338754189#gid=338754189&range=B32:B34) isn't tested. Would it make sense test this as well?

Reply:
You're right. I've enabled `testMedianReaders` to test this instruction